resources/user-agents/browsers/yabrowser/yabrowser-1-20-14-12.json

```diff
@@ -269,6 +269,68 @@
           ]
         }
       ]
+    },
+    {
+      "userAgent": "Yandex Browser #MAJORVER#.#MINORVER# for iOS",
```

Comment:
Does for the iOS version exist the same version range? Or should this moved into a separate file?